src/IFrameNavigator.ts

```diff
@@ -107,6 +103,7 @@ export default class IFrameNavigator implements Navigator {
     private scroller: ScrollingBookView | null;
     private annotator: Annotator | null;
     private settings: BookSettings;
+    private eventHandler: EventHandler | null;
```

Comment:
This is declared nullable but it looks like it's always initialized to at least `new EventHandler()`, it seems. Perhaps the `null` case can be removed here.